Resources/Prototypes/Entities/Objects/Tools/cowtools.yml

```diff
@@ -147,6 +147,9 @@
     - key: enum.GasAnalyzerUiKey.Key
       type: GasAnalyzerBoundUserInterface
   - type: Appearance
+  - type: Tag
```

Comment:
You can whitelist already existing component GasAnalyzer instead of adding a new tag

Reply:
Looks like this was mixed with your other PR but yeah agree, just use the components for the whitelist.

Reply:
i use tag because there two thing what can analyze gas
for tray i changed to component